pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -18,16 +21,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ python makeWrapper zip pandoc ];
 
-  patchPhase = "rm youtube-dl";
-
-  configurePhase = ''
-    makeFlagsArray=( PREFIX=$out SYSCONFDIR=$out/etc PYTHON=${python}/bin/python )
```

Comment:
that also appears to be the default, using $prefix/etc as the sysconfdir, so it shouldn't be needed if we don't use make anywhere, which the python version of this package doesn't.